src/Markdown.ts

```diff
@@ -27,8 +29,11 @@ const TEXT_NODES = ['text', 'softbreak', 'linebreak', 'paragraph', 'document'];
 interface CommonmarkHtmlRendererInternal extends commonmark.HtmlRenderer {
     paragraph: (node: commonmark.Node, entering: boolean) => void;
     link: (node: commonmark.Node, entering: boolean) => void;
-    html_inline: (node: commonmark.Node) => void; // eslint-disable-line camelcase
+    html_inline: (node: commonmark.Node) => void; // eslint-disable-line camelcazse
```

Comment:
er?

Reply:
Rebasing and splitting things into smaller commits from a bigger branch, must have accidentally commited that